core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -77,24 +77,19 @@
   public static LocalCacheManager create(AlluxioConfiguration conf) throws IOException {
     MetaStore metaStore = MetaStore.create();
     CacheEvictor evictor = CacheEvictor.create(conf);
-    PageStore pageStore = PageStore.create(conf);
+    PageStoreOptions options = PageStoreOptions.create(conf);
+    PageStore pageStore = null;
     try {
-      Collection<PageInfo> pageInfos = pageStore.getPages();
-      LOG.info("Creating LocalCacheManager with {} existing pages", pageInfos.size());
-      for (PageInfo pageInfo : pageInfos) {
-        PageId pageId = pageInfo.getPageId();
-        metaStore.addPage(pageId, pageInfo);
-        evictor.updateOnPut(pageId);
-      }
-      return new LocalCacheManager(conf, metaStore, pageStore, evictor);
+      pageStore = PageStore.create(options, metaStore, evictor);
     } catch (Exception e) {
-      try {
-        pageStore.close();
-      } catch (Exception ex) {
-        e.addSuppressed(ex);
-      }
-      throw new IOException("failed to create local cache manager", e);
+      LOG.error("Failed to restore PageStore", e);
```

Comment:
After restoration failed, we will need to clean the store directory before creating a new store.

Reply:
in this case, passing `init=true` to create PageStore in the following.

Reply:
The current `initialize()` function does not seem to clean up the store path of the current store type.

Reply:
fixed